interface/modules/custom_modules/oe-module-weno/templates/indexrx.php

```diff
@@ -0,0 +1,137 @@
+$facility_name = $wenoProperties->getFacilityInfo();
+
+$newRxUrl = "https://online.wenoexchange.com/en/NewRx/ComposeRx?useremail=";
+if ($urlParam == 'error') {   //check to make sure there were no errors
+    echo xlt("Cipher failure check encryption key");
+    exit;
+}
+?>
+
+<style>
+    .row {
+        display: flex;
+        flex-direction: row;
+    }
+    .col {
+        display: flex;
+        flex-direction: column;
+    }
+    .mr-5 {
+        margin-right: 50px;
+    }
+    .center {
+        justify-content: center;
+    }
+</style>
+
+<!doctype html>
+<html lang="en">
+<head>
+ <meta charset="utf-8">
+ <title><?php echo xlt('Weno eRx') ?></title>
+</head>
+<body >
+     
+<?php
+    //**warning** do not add urlencode to  $provider_info['email']
+    $urlOut = $newRxUrl . $provider_info['email'] . "&data=" . urlencode($urlParam);
+
+?>
+
+<div style="margin:30px">
+    <div>
+        <h2><?php echo xlt("e-Prescribe"); ?></h2>
+    </div>
+    <div class="container">
+        <div class="row center">
+            <div class="col mr-5">
+                <div class="row">
+                    <div><b><?php echo xlt("Presriber"); ?></b>: <?php echo text($provider_name); ?> </div>
+                </div>
+                <div class="row">
+                    <div><b><?php echo xlt("Patient Name"); ?></b>: <?php echo text($patient_name) ?></div>
+                </div>
+                <div class="row">
+                    <div>
+                        <b><?php echo xlt("Current Facility/Location"); ?></b>: <?php echo text($facility_name["name"])?>
+                    </div>
+                </div>
+            </div>
+            <div class="col mr-5" style="margin-top: -14px !important">
+                <table>
+                    <tr>
+                        <th><?php echo xlt("Vitals"); ?></th>
+                        <th><?php echo xlt("Date Observed"); ?></th>
+                    </tr>
+                    <tr>
+                        <td><?php echo xlt("Height"); ?>:<?php echo number_format($vitals['height'], 2) ?> </td>
+                        <td><?php echo oeFormatShortDate(date("Y-m-d", strtotime($vitals['date'])))?></td>
+                    </tr>
+                    <tr>
+                        <td><?php echo xlt("Weight: "); ?><?php echo number_format($vitals['weight'], 2)?> </td>
+                        <td><?php echo oeFormatShortDate(date("Y-m-d", strtotime($vitals['date'])))?></td>
+                    </tr>
+                </table>
+            </div>
+            <div class="col">
+                <div><?php echo xlt("Primary Pharmacy: "); ?> <?php echo xlt($prim_pharmacy['business_name'] . " / " . $primary_pharmacy['address_line_1'] . " / " . $primary_pharmacy['city']); ?></div>
+                <div><?php echo xlt("Weno Alt: "); ?> <?php echo xlt($alt_pharmacy['business_name'] . " / " . $pharmacy['address_line_1'] . " / " . $pharmacy['city']); ?></div>
```

Comment:
For both lines above:
1. No trailing spaces or trailing colons in translations contants
2. Do not translate business names and addresses
Here is fixed up code:
```
                <div><?php echo xlt("Primary Pharmacy"); ?> : <?php echo text($prim_pharmacy['business_name'] . " / " . $primary_pharmacy['address_line_1'] . " / " . $primary_pharmacy['city']); ?></div>
                <div><?php echo xlt("Weno Alt"); ?> : <?php echo text($alt_pharmacy['business_name'] . " / " . $pharmacy['address_line_1'] . " / " . $pharmacy['city']); ?></div>
```